src/cultionet/models/unet_parts.py

```diff
@@ -1,14 +1,19 @@
 import typing as T
+import enum
 
+import torch
+
+from . import model_utils
 from .base_layers import (
+    AttentionGate,
+    AtrousPyramidPooling,
     DoubleConv,
     PoolConv,
     PoolResidualConv,
+    ResidualAConv,
     ResidualConv
 )
-from . import model_utils
-
-import torch
+from .enums import ModelTypes, ResBlockTypes
 
 
 class UNet3Connector(torch.nn.Module):

```

Comment:
This is the UNet class that allows for connecting any number of UNet3 connections.